py-polars/docs/source/reference/series/temporal.rst

```diff
@@ -16,34 +16,34 @@ The following methods are available under the `Series.dt` attribute.
     Series.dt.date
     Series.dt.datetime
     Series.dt.day
-    Series.dt.days
+    Series.dt.total_days
```

Comment:
The old methods should remain part of the API reference until they are no longer available.